corgi/collectors/brew.py

```diff
@@ -354,75 +354,129 @@ def get_container_build_data(self, build_id: int, build_info: dict) -> dict:
 
             component["meta"]["build_parent_nvrs"] = build_parent_nvrs
 
-        source_components: list[dict[str, Any]] = []
+        # remote_source json, and tar download urls by cachito url
+        remote_sources: dict[str, Tuple] = {}
+
+        # builds such as 1911112 have all their info in typeinfo as they use remote_sources map in
```

Comment:
nice to keep all this logic in the collector - appreciating the code comments explaining rationale